index.html

```diff
@@ -1112,6 +1112,71 @@ <h2>Thing Description Server</h2>
                     </p>
                 </dd>
             </dl>
+            <dl>
+                <dt>CoAP</dt>
+                <dd>
+                <p>
+                    <span class="rfc2119-assertion" id="service-coap-secure">
+                        A CoAP-based TD Server providing a <a>TD</a> SHOULD use the appropriate transport
+                        security mechanism for the transport protocol used (i.e., DTLS for CoAP over UDP
+                        or TLS for CoAP over TCP).
```

Comment:
I wasn't sure if OSCORE should be mentioned here as an alternative/additional method for securing CoAP messages (CC @j1y3p4rk).

Reply:
That simplifies things a lot :) Removed the assertions in 35d7b3c5af7742a76336d3b58a6f82a296c5d4c3.